With opaque EVP_MD_METHOD, erlang needs to use EVP_MD_meth_*()
instead of initializing the method statically.

Index: lib/crypto/c_src/otp_test_engine.c
--- lib/crypto/c_src/otp_test_engine.c.orig
+++ lib/crypto/c_src/otp_test_engine.c
@@ -33,8 +33,7 @@
 #define PACKED_OPENSSL_VERSION_PLAIN(MAJ, MIN, FIX) \
     PACKED_OPENSSL_VERSION(MAJ,MIN,FIX,('a'-1))
 
-#if OPENSSL_VERSION_NUMBER < PACKED_OPENSSL_VERSION_PLAIN(1,1,0) \
-    || defined(LIBRESSL_VERSION_NUMBER)
+#if OPENSSL_VERSION_NUMBER < PACKED_OPENSSL_VERSION_PLAIN(1,1,0)
 # define OLD
 #endif
 
